internal/commands/image/list_test.go

```diff
@@ -26,10 +26,10 @@ type bundleStoreStubForListCmd struct {
 	refList []reference.Reference
 }
 
-func (b *bundleStoreStubForListCmd) Store(ref reference.Reference, bndle *bundle.Bundle) (reference.Reference, error) {
+func (b *bundleStoreStubForListCmd) Store(ref reference.Reference, bndle *bundle.Bundle) (reference.Digested, error) {
 	b.refMap[ref] = bndle
 	b.refList = append(b.refList, ref)
-	return ref, nil
+	return store.FromBundle(bndle)
```

Comment:
I know it's not part of this change but those method are really unclear for me.
Should be something like `store.IDFromBundle(bundle)`, `store.FromBundle` make me think we're retrieving a store from a bundle.

Reply:
PRs welcome ;)